apps/redi-talent-pool/src/pages/app/browse/BrowseJobseeker.tsx

```diff
@@ -145,6 +206,17 @@ export function BrowseJobseeker() {
                 }
               />
             ))}
+            {/* TODO: Uncomment the next block when we have federal states for joblistings */}
```

Comment:
Perhaps create a ticket for this?

Reply:
I will actually ask Miriam about this in tne next meeting first, then decide what to do with this filter.

Reply:
I guess we can resolve this convo?

Reply:
Yes, created #550 to cover this!